### PR TITLE
Update flow to get token for labapi to not require automationrunner app

### PR DESCRIPTION
**Pull request by iamgusain**
### What
Updating the flow to get token for labapi 

### Why
We are moving the steps to call labapi in our test libs
from
AutomationRunnerApp Cred --> AutomatoinRunnerApp AT --> MSIDLAB app Creds --> MSIDLAB app AT --> LabApi

to
MSIDLAB app Creds --> MSIDLAB app AT --> LabApi

### How
Updated code to remove the first two steps in the flow

### Testing
Verified running the test locally and in pipeline.

### CODE REVIEW

**Review by melissaahn — APPROVED** [review]

**Review by fadidurah — APPROVED** [review]

File: azure-pipelines/templates/steps/automation-cert.yml [hunk 1]
```diff
@@ -10,18 +10,18 @@ steps:
...
-- task: AzureKeyVault@1
+- task: AzureKeyVault@2
   displayName: 'Azure Key Vault: Download Cert for Automation'
   inputs:
-    ConnectedServiceName: 9a9308ae-59b4-475f-b49c-3e828c508d8b
-    KeyVaultName: ADALTestInfo
-    SecretsFilter: AndroidAutomationRunner
+    azureSubscription: 'AuthSdkResourceManager'
+    KeyVaultName: 'buildautomation'
+    SecretsFilter: 'AzureADIdentityDivisionTestAgentCert'
```
Comment (shahzaibj) [inline azure-pipelines/templates/steps/automation-cert.yml]: I'm not familiar with this. Is this a new cert? What is this used for?

File: LabApiUtilities/src/test/com/microsoft/identity/labapi/utilties/authentication/common/MicrosoftClientAssertionTest.java [hunk 2]
```diff
@@ -42,6 +43,7 @@ public class MicrosoftClientAssertionTest {
     private final static String KEYSTORE_PROVIDER = "SunMSCAPI";
     private final static String AUDIENCE = "some_audience";
 
+    @Ignore
```
Comment (shahzaibj) [inline LabApiUtilities/src/test/com/microsoft/identity/labapi/utilties/authentication/common/MicrosoftClientAssertionTest.java]: I believe this test is independent of the changes you are making and test likely still valid

File: LabApiUtilities/src/test/com/microsoft/identity/labapi/utilties/authentication/common/CertificateCredentialTest.java [hunk 3]
```diff
@@ -214,6 +215,7 @@ public byte[] getExtensionValue(String oid) {
         Assert.assertNotNull(certificateCredential.getPublicCertificate());
     }
 
+    @Ignore
```
Comment (shahzaibj) [inline LabApiUtilities/src/test/com/microsoft/identity/labapi/utilties/authentication/common/CertificateCredentialTest.java]: I believe this test is independent of the changes you are making and test likely still valid

File: LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/authentication/LabApiAuthenticationClient.java [hunk 4]
```diff
@@ -37,28 +33,25 @@
...
+    private final static String CLIENT_ID = "f62c5ae3-bf3a-4af5-afa8-a68b800396e9";
+    private final String mClientSecret;
 
     public LabApiAuthenticationClient(@NonNull final IConfidentialAuthClient confidentialAuthClient) {
         mConfidentialAuthClient = confidentialAuthClient;
-        mKeyVaultAuthenticationClient = new KeyVaultAuthenticationClient(confidentialAuthClient);
+        mClientSecret = "";
     }
 
     public LabApiAuthenticationClient(@NonNull final IConfidentialAuthClient confidentialAuthClient,
```
Comment (shahzaibj) [inline LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/authentication/LabApiAuthenticationClient.java:47]: Do we still need `confidentialAuthClient` as input to the constructors for this class?

**Comment by somalaya** [comment]
Looks like we need to also update instructions for running the UI tests in local?

File: azure-pipelines/templates/steps/automation-cert.yml (hunk 1 above)
Comment (iamgusain) [reply azure-pipelines/templates/steps/automation-cert.yml]: reverted

File: LabApiUtilities/src/test/com/microsoft/identity/labapi/utilties/authentication/common/MicrosoftClientAssertionTest.java (hunk 2 above)
Comment (iamgusain) [reply LabApiUtilities/src/test/com/microsoft/identity/labapi/utilties/authentication/common/MicrosoftClientAssertionTest.java]: reverted

File: LabApiUtilities/src/test/com/microsoft/identity/labapi/utilties/authentication/common/CertificateCredentialTest.java (hunk 3 above)
Comment (iamgusain) [reply LabApiUtilities/src/test/com/microsoft/identity/labapi/utilties/authentication/common/CertificateCredentialTest.java]: reverted

File: LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/authentication/LabApiAuthenticationClient.java (hunk 4 above)
Comment (iamgusain) [reply LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/authentication/LabApiAuthenticationClient.java:47]: removed

**Comment by iamgusain** [comment]
> Looks like we need to also update instructions for running the UI tests in local?

Updating the docs in this PR https://github.com/AzureAD/android-complete/pull/310

**Pull request merged**